kanidmd/src/lib/valueset.rs

```diff
@@ -0,0 +1,210 @@
+use crate::prelude::*;
+// use hashbrown::HashSet;
+use std::borrow::Borrow;
+use std::collections::BTreeSet;
+use std::iter::FromIterator;
+
+pub struct ValueSet {
+    inner: BTreeSet<Value>,
+}
+
+impl Default for ValueSet {
+    fn default() -> Self {
+        ValueSet {
+            inner: BTreeSet::new(),
+        }
+    }
+}
```

Comment:
```suggestion
#[derive(Default)]
pub struct ValueSet {
    inner: BTreeSet<Value>,
}
```

Reply:
See before - this is about stubs so that when we strongly type we can "fill in the gaps".